server/config/firebaseAdmin.ts

```diff
@@ -0,0 +1,20 @@
+import admin from 'firebase-admin';
+import * as serviceAccount from './serviceAccount.json';
+
+const params: object = {
+  type: serviceAccount.type,
+  project_id: serviceAccount.project_id,
+  private_key_id: serviceAccount.private_key_id,
+  private_key: serviceAccount.private_key,
+  client_email: serviceAccount.client_email,
+  client_id: serviceAccount.client_id,
+  auth_uri: serviceAccount.auth_uri,
+  token_uri: serviceAccount.token_uri,
+  auth_provider_x509_cert_url: serviceAccount.auth_provider_x509_cert_url,
+  client_x509_cert_url: serviceAccount.client_x509_cert_url,
+  universe_domain: serviceAccount.universe_domain,
+};
+
```

Comment:
is this file initializing firebase for the backend? Do we not need to use the api key here? I could use some comments to understand what this file is doing

Reply:
Hi Lucie. It's quiet confusing between the setup for frontend and backend. 
This is the configuration for Firebase Admin, which uses the serviceAccount key that needs to be downloaded from the Firebase console (only authorized members will have access, and this is highly confidential. PS. The serviceAccount key is the only key that can be regenerated). On the other hand, the apiKey is what we use to connect our web application to our Firebase project, which is less sensitive.